src/generator-add-web-assets.js

```diff
@@ -130,10 +130,12 @@ class ExtensionWebAssetsGenerator extends Generator {
         customActions.forEach((action) => {
             if (action.needsModal) {
                 const modalComponentName = action.componentName;
+                const modalType = action.modalType;
                 this.fs.copyTpl(
                     this.templatePath(relativeTemplatePath),
                     this.destinationPath(path.join(this.destFolder, `./src/components/${modalComponentName}.js`)), {
-                        ModalComponentName: modalComponentName
+                        ModalComponentName: modalComponentName,
+                        ModalType: modalType,
```

Comment:
given that `ModalComponentName` and `ModalType` are just local JS variables inside ejs template, would it be more appropriate to use camel case here?